lib/services/runtime/handlers/message/lib/conditions/script.condition.ts

```diff
@@ -1,10 +1,10 @@
 import { CompiledScriptCondition } from '@voiceflow/dtos';
 
+import { IVMIsolate } from '../../../../../../../runtime/lib/Handlers/utils/ivmIsolate';
```

Comment:
use '@' to meet repo root (as set in tsconfig)
```suggestion
import { IVMIsolate } from '@/runtime/lib/Handlers/utils/ivmIsolate';
```

Reply:
Automatically rewrite imports after moving file messed up the transformation